e2e_tests/tests/command/test_tensorboard.py

```diff
@@ -159,3 +161,48 @@ def test_start_tensorboard_for_multi_experiment(tmp_path: Path, secrets: Dict[st
                 break
         else:
             raise AssertionError(f"Did not find {SERVICE_READY} in output")
+
+
+@pytest.mark.slow
+@pytest.mark.e2e_cpu
+def test_start_tensorboard_with_custom_image(tmp_path: Path) -> None:
+    """
+    Start a random experiment configured with the shared_fs backend, start a
+    TensorBoard instance pointed to the experiment with custom image, verify
+    the image has been set, and kill the TensorBoard instance.
+    """
+    with FileTree(tmp_path, {"config.yaml": shared_fs_config(1)}) as tree:
+        config_path = tree.joinpath("config.yaml")
+        experiment_id = exp.run_basic_test(
+            str(config_path), conf.fixtures_path("no_op"), num_trials
+        )
```

Comment:
Couldn't all of this just be:

```python
experiment_id = exp.run_basic_test(
    conf.fixtures_path("no_op/single-one-short-step.yaml"),
    conf.fixtures_path("no_op"),
    1,
)
```

I see that the `shared_fs_config` function in this file is from the old pedl repo (that is, it's very old) but I think it's just tech debt, that it doesn't add anything, and you should avoid it here.